code/game/turfs/turf.dm

```diff
@@ -27,6 +27,7 @@
 	flags = 0
 
 	var/image/obscured	//camerachunks
+	var/datum/weather/weather//Is there active weather on this turf?
```

Comment:
don't actually think we need this reference OR the loop finding the turfs anymore.
```/area``` could do with a weather reference now though, considering it's area based.